lib/ReactViewModels/ViewState.ts

```diff
@@ -311,7 +311,8 @@ export default class ViewState {
    * Gets or sets a value indicating whether the catalog's modal share panel
    * is currently visible.
    */
-  @observable shareModalIsVisible: boolean = false;
+  @observable shareModalIsVisible: boolean = false; // Small share modal inside StoryEditor
+  @observable retainSharePanel: boolean = false;
```

Comment:
Might be good to add a line of comment documenting the observable.

Reply:
Done